tests/client/client_cache_manager_unit_test.cc

```diff
@@ -0,0 +1,175 @@
+#include "gtest/gtest.h"
+
+#include "absl/time/time.h"
+#include "src/client/client_cache_manager.h"
+
+using namespace gfs::client;
+
+// Helper function to easily construct a protos::ChunkServerLocation object
+// by passing the host name and port number. In a sense, this function 
+// serves as a constructor for the designated type with non-trivial
+// parameters, since proto generates only a default constructor
+protos::ChunkServerLocation ChunkServerLocationBuilder(
+    const std::string& hostname, const uint32_t port) {
+  protos::ChunkServerLocation location;
+  location.set_server_hostname(hostname);
+  location.set_server_port(port);
+  return location; 
+}
```

Comment:
nit: want to make this an utility method for reuse? Instead of copying

Reply:
Yes, let me refactor this in a followup.